pycobertura/cli.py

```diff
@@ -95,8 +95,22 @@ def get_exit_code(differ, source):
     "the --source is a zip archive and the files were zipped under "
     "a directory prefix that is not part of the source.",
 )
+@click.option(
+    "-hide",
+    "--hide-columns",
+    default="",
+    type=str,
+    help="Comma-separated string with columns you want to hide",
```

Comment:
```suggestion
    help="Comma-separated list of columns names you want to hide",
```

Reply:
done in https://github.com/aconrad/pycobertura/pull/160/commits/805ceff720a025ed5e9247d0925c63dbf4c650d6